lib/cart/cart_repository.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flex_storefront/cart/apis/cart_api.dart';
+import 'package:flex_storefront/cart/models/cart.dart';
+import 'package:flex_storefront/cart/models/cart_message.dart';
+import 'package:rxdart/subjects.dart';
+
+const kTestCart = '1b2be4d8-c5c7-4b0a-a472-ac7df417d762';
+
+class CartRepository {
+  CartRepository({
+    required CartApi cartApi,
+  }) : _cartApi = cartApi {
+    init();
+  }
+
+  final CartApi _cartApi;
+
+  late final _cartStreamController = BehaviorSubject<Cart>();
```

Comment:
I'm wondering if it's necessary to bring rxdart into the mix 🤔 Can't we simply instanciate a `StreamController<Cart>()`? (from `dart:async`)